README.md

```diff
@@ -0,0 +1,101 @@
+# Releasing New Documentation Version
+
+## Adding docs for a new release
+
+Official docs for a release are built and pushed when a final tag
+is created via GitHub. When -rc tags are built, you should see a directory
+named after the tag with the -rc suffix and the prefix "v"
+truncated. For example, v0.1.0-rc1 would become just 0.1. This way
+you can preview your documentation before the release.
+
+After the final tag is created, an action will move all the docs from
+`release/xx` branch to a directory named after your release version.
+
+When an -rc tag dir is pushed to the gh-pages branch, update versions.html as follows:
+
+```
+<li class="toctree-l1">
+  <a class="reference internal" href="0.1/">main (unstable)</a>
+</li>
+<li class="toctree-l1">
+  <a class="reference internal" href="0.1/">v0.2.0 (release candidate)</a> - replace `v0.2.0` with your new tag
+</li>
+<li class="toctree-l1">
+  <a class="reference internal" href="0.1/">v0.1.0 (stable)</a>
+</li>
+... 
+```
+
+### Updating `stable` versions
+
+The stable directory is a symlink to the latest released version.
+On the day of the release, you need to update the symlink to the
+release version. For example:
+
+```
+git checkout gh-pages
+rm stable # remove the existing symlink. **Do not** edit!
+ln -s 0.1 stable   # substitute the correct version number here
+git add stable
+git commit -m "Update stable to 0.1"
+git push -u origin
```

Comment:
Rather than force-pushing, please ask the reader to create a PR like you do below: "Submit a PR and merge into the `gh-pages` branch." I'd like people to create PRs and have them approved before merging.